internal/cluster/transition.go

```diff
@@ -577,6 +584,11 @@ func (th *transitionHandler) PostRefreshCluster(reason string) stateswitch.PostT
 		if !ok {
 			return errors.New("PostRefreshCluster invalid argument")
 		}
+		var values []interface{}
+		for _, p := range fargs {
+			values = append(values, p(sCluster))
+		}
+		reason := fmt.Sprintf(template, values...)
```

Comment:
This is clever, but can we give the variables better names to make this a bit more understandable?
Maybe: something like `clusterFormatFunc` instead of `p`?

Reply:
OK